### PR TITLE
chore: add space on right for small screen scrolling

### PR DESCRIPTION
**Pull request by cmotelet**
In mode design, on a small touchscreen when you try to scroll the screen often you drag the widget rather than scroll the screen.
There is now a little space (20px) on the right for scrolling.

### CODE REVIEW

File: lib/ui/main.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import BaconModel from 'bacon.model';
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
-import RGL, { WidthProvider } from "react-grid-layout";
+import RGL from "react-grid-layout";
```
Comment (tkurki) [inline lib/ui/main.js]: Why do we need our own copy?

File: package.json [hunk 2]
```diff
@@ -31,12 +31,15 @@
   },
   "devDependencies": {
     "@signalk/client": "^2.0.4",
-    "@babel/core": "^7.7.2",
-    "@babel/preset-env": "^7.7.1",
-    "@babel/register": "^7.7.0",
-    "@babel/preset-react": "^7.7.0",
+    "@babel/core": "^7.8.4",
```
Comment (tkurki) [inline package.json]: Maybe next time put updates to existing deps in a separate commit, so that the additions that go with the feature are easier to distinguish.

File: lib/ui/main.js (hunk 1 above)
Comment (cmotelet) [reply lib/ui/main.js]: WidthProvider is provided in react-grid-layout as a template for a minimal implementation.
https://github.com/STRML/react-grid-layout#providing-grid-width.
I need to shrink the grid when in edit mode to be able to scroll more easily.
image
image

File: package.json (hunk 2 above)
Comment (cmotelet) [reply package.json]: I needed an upgrade so that I could transplile some jsx file (WidthProvider.jsx)
